lib/public/views/Logs/ActiveColumns/logsActiveColumns.js

```diff
@@ -49,6 +49,33 @@ export const logsActiveColumns = {
         primary: true,
         profiles: [profiles.none, 'embeded'],
     },
+    view: {
+        name: '',
+        visible: true,
+        sortable: false,
+        format: (_, log) => h(
+            '.w-100',
+            frontLink(
+                'View',
+                'log-detail',
+                { id: log.id },
+                {
+                    id: `btn${log.id}`,
+                    class: 'btn btn-primary btn-redirect w-100',
+                    style: {
+                        paddingLeft: '0.5rem',
+                        paddingRight: '0.5rem',
+                    },
+                },
+            ),
+        ),
+        profiles: {
+            embeded: true,
+            home: {
+                size: 'w-10',
+            },
+        },
+    },
```

Comment:
Please have a look at the other active columns to see how it is done.
For example, do not use inline styles, name the column `actions`.